fhir-server/liberty-config/configDropins/defaults/bulkdata.xml

```diff
@@ -61,5 +61,5 @@
     -->
 
     <batchPersistence jobStoreRef="BatchDatabaseStore"/>
-    <databaseStore dataSourceRef="fhirbatchDS" id="BatchDatabaseStore" schema="JBATCH" tablePrefix=""/>
+    <databaseStore dataSourceRef="fhirbatchDS" id="BatchDatabaseStore" schema="FHIR_BATCH" tablePrefix="" createTables="false"/>
```

Comment:
better to use FHIR_JBATCH, because JBATCH is the default schema of javabatch.

Reply:
does it make a difference?

Reply:
Converted to JBatch